browser/utils.js

```diff
@@ -9,7 +9,7 @@ export function putUserOnDOM (user) {
     const head = document.createElement('a-minecraft');
     scene.appendChild(head);
     head.setAttribute('id', user.id);
-    head.setAttribute('minecraft-nickname', user.color);
+    head.setAttribute('minecraft-nickname', user.displayName);
```

Comment:
I think revert this back, 1. @Jmikeydarby is working on this presently and 2. this doesn't work how we originally thought it did haha.